packages/extras/src/lib/components/Environment/Environment.svelte.d.ts

```diff
@@ -0,0 +1,36 @@
+import type { Scale } from '@threlte/core'
```

Comment:
This type will change in the transition to v5 and will probably just vanish. Can you change the type of this prop to something more v5-ish?

Reply:
Something like this? 

`[scale: number | { x?: number; y?: number; z?: number } | number[]](scale: number | Partial<Vector3> | number[])`

The Scale type is actually pretty handy for these things but I get why we're tossing it.